ledger/src/blockstore_processor.rs

```diff
@@ -624,90 +633,125 @@ fn process_entries(
                     process_batches(
                         bank,
                         replay_tx_thread_pool,
-                        &batches,
+                        batches.drain(..),
                         transaction_status_sender,
                         replay_vote_sender,
                         batch_timing,
                         log_messages_bytes_limit,
                         prioritization_fee_cache,
                     )?;
-                    batches.clear();
                     for hash in &tick_hashes {
                         bank.register_tick(hash);
                     }
                     tick_hashes.clear();
```

Comment:
nit: `for hash in tick_hashes.drain(..)` for the api use pattern consistency with the `batches` vec?

Reply:
56d20aa95e93c2a464fbd2d9b9e419ae6256940e